Makefile

```diff
@@ -1,7 +1,11 @@
 # These variables get inserted into ./build/commit.go
-BUILD_TIME=$(shell date)
 GIT_REVISION=$(shell git rev-parse --short HEAD)
 GIT_DIRTY=$(shell git diff-index --quiet HEAD -- || echo "✗-")
+ifeq ("$(GIT_DIRTY)", "✗-")
+	BUILD_TIME=$(shell date)
+else
+	BUILD_TIME=$(shell git show -s --format=%ci HEAD)
```

Comment:
Does this set the build time to the last commit timestamp? Is that desirable?

Reply:
It’s what release.sh did, I think it was for more reproducible binaries.